awx_collection/plugins/modules/host.py

```diff
@@ -43,6 +43,7 @@
       description:
         - If the host should be enabled.
       type: bool
+      default: true
```

Comment:
This is going to set the `enabled` flag to true for any existing host where the option isn't specified.

Reply:
you are right..........................................................................................................................................................................................................................................................................................................................................................................................................................................................